contracts/LosslessControllerV1.sol

```diff
@@ -26,42 +26,48 @@ contract LosslessControllerV1 is Initializable, ContextUpgradeable, PausableUpgr
         _;
```

Comment:
Maybe let's not modify this contract cause it's already deployed.

Reply:
Reverted!